src/model/collection/util.ts

```diff
@@ -1,6 +1,10 @@
 /**
- * @param folder - the folder to sanitize.
- * @returns sanitized value with trailing slashes removed, if applicable.
+ * Returns a sanitized folder path by removing a trailing slash unless the path is the root ("/").
+ *
+ * The function trims whitespace from the input folder string. If the trimmed string equals "/", it is returned unchanged. Otherwise, any trailing slash is removed.
+ *
+ * @param folder - The folder path to sanitize.
+ * @returns The folder path without a trailing slash, except for the root path.
```

Comment:
```suggestion
 * @param folder - The folder path to sanitize.
 * @returns The folder path without a trailing slash, except for the root path.
```
The project's JSDoc ESLint rules flags extra lines between tags as errors.